access_list.go

```diff
@@ -115,12 +103,12 @@ func (acl *AccessListEntry) GetValues() string {
 }
 
 // IsClaimAllowed checks whether access list entry allows the claims.
-func (acl *AccessListEntry) IsClaimAllowed(claims *UserClaims) (bool, bool) {
+func (acl *AccessListEntry) IsClaimAllowed(claims *UserClaims) bool {
```

Comment:
@njones , please match the changes I made ... while you were working on your PR, I added ACL changes. It works a bit different. I documented it in `README.md`.